test/ImpactEvaluator.t.sol

```diff
@@ -230,6 +230,149 @@ contract ImpactEvaluatorTest is Test {
         impactEvaluator.setScores(0, addresses, scores);
     }
 
+    function test_SetScoresAfterDecreaseMinBalanceForTransfer() public {
+        ImpactEvaluator impactEvaluator = new ImpactEvaluator(address(this));
+        vm.deal(payable(address(impactEvaluator)), 100 ether);
+
+        impactEvaluator.adminAdvanceRound();
+        impactEvaluator.adminAdvanceRound();
+
+        address payable[] memory addresses = new address payable[](3);
+        addresses[0] = payable(vm.addr(1));
+        addresses[1] = payable(vm.addr(2));
+        addresses[2] = payable(vm.addr(3));
+        uint[] memory scores = new uint[](3);
+        scores[0] = 1;
+        scores[1] = impactEvaluator.MAX_SCORE() - scores[0];
+        scores[2] = 0;
+
+        impactEvaluator.setScores(1, addresses, scores);
+        assertEq(
+            impactEvaluator.participantIsReadyForTransfer(vm.addr(1)),
+            false,
+            "participant 0 is not ready"
+        );
+        assertEq(
+            impactEvaluator.participantIsReadyForTransfer(vm.addr(2)),
+            true,
+            "participant 1 is ready"
+        );
+        assertEq(
+            impactEvaluator.participantIsReadyForTransfer(vm.addr(3)),
+            false,
+            "participant 2 is not ready"
+        );
+
+        impactEvaluator.setMinBalanceForTransfer(1);
+        impactEvaluator.adminAdvanceRound();
+
+        address payable[] memory addresses2 = new address payable[](4);
+        addresses2[0] = payable(vm.addr(1));
+        addresses2[1] = payable(vm.addr(2));
+        addresses2[2] = payable(vm.addr(3));
+        addresses2[3] = payable(0x000000000000000000000000000000000000dEaD);
+        uint[] memory scores2 = new uint[](4);
+        scores2[0] = 0;
+        scores2[1] = 0;
+        scores2[2] = 0;
+        scores2[3] = impactEvaluator.MAX_SCORE();
+        impactEvaluator.setScores(2, addresses2, scores2);
+        assertEq(
+            impactEvaluator.participantIsReadyForTransfer(vm.addr(1)),
+            true,
+            "participant 0 is now ready"
+        );
+        assertEq(
+            impactEvaluator.participantIsReadyForTransfer(vm.addr(2)),
+            true,
+            "participant 1 is still ready"
+        );
+        assertEq(
+            impactEvaluator.participantIsReadyForTransfer(vm.addr(3)),
+            false,
+            "participant 2 is still not ready"
+        );
+        assertEq(impactEvaluator.readyForTransfer(0), vm.addr(2));
+        assertEq(impactEvaluator.readyForTransfer(1), vm.addr(1));
+        vm.expectRevert();
+        impactEvaluator.readyForTransfer(2);
```

Comment:
What do these two lines do?

Shouldn't we assert that `impactEvaluator.readyForTransfer(2)` returns `false`?

Reply:
If you are trying to assert that there are only 2 participants ready for transfer, then I think the following check is easier to understand and probably produces more helpful error message too.

```solidity
assertEq(impactEvaluator.participantCountReadyForTransfer(), 2)
```

See https://github.com/Meridian-IE/impact-evaluator/blob/087c50d401ed7d67de0229bc227f0149764d0671/src/Balances.sol#L21-L23

Alternatively:

```solidity
assertEq(impactEvaluator.readyForTransfer.length, 2)
```

Reply:
> What do these two lines do?
>
> Shouldn't we assert that impactEvaluator.readyForTransfer(2) returns false?

Context is important: The line above tells the compiler that the next line should throw. This function is none that we provided, it's how Solidity exposes public arrays to contract consumers. It's effectively `return array[index]` but throws if `index` can't be found.

Here we want to assert how many participants are ready for transfer, and what their addresses are.

Reply:
> Here we want to assert how many participants are ready for transfer, and what their addresses are.


Yes, that was my assumption too.

I am trying to say that the following code makes your intent more explicit / easier to understand, especially for people unfamiliar with solidity testing, like me.

```solidity
assertEq(impactEvaluator.readyForTransfer.length, 2)
```

Compare that with your current version

```solidity
vm.expectRevert();
impactEvaluator.readyForTransfer(2);
```

Anyhow, this is not a big deal; feel free to keep what you have now.

Reply:
> assertEq(impactEvaluator.readyForTransfer.length, 2)

This one isn't available in array getters, you can only get an element at an index